### ISSUE TITLE
Issue with Mongol 

### ISSUE THREAD

**Issue by msavin** [JackAdams/meteor-editable-json#3]
Hey man,

For some reason, its returning "No Documents Found" on all collections.  I was able to replicate it by cloning the todos app and installing Mongol. It'd be great if we can fix it asap because I think for the past week people have been downloading a broken version.

PS: still waiting for your email :)

```
[Log] Exception from Tracker recompute function: 'undefined' is not an object (evaluating 'Collection.find') (meteor.js, line 888)
http://localhost:4000/packages/babrahams_editable-json.js?1047754ffc6951042e6ae766224ac96170c70dc1:560:27
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
Template.editableJSON:anonymous@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
Computation@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:224:18
autorun@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:499:34
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1831:29
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3267:27
created@http://localhost:4000/packages/babrahams_editable-json.js?1047754ffc6951042e6ae766224ac96170c70dc1:558:17
runTemplateHooks@http://localhost:4000/packages/aldeed_template-extension.js?e21d2f3f76cf4d629c394319bdb4c45f79da30f1:403:15
http://localhost:4000/packages/aldeed_template-extension.js?e21d2f3f76cf4d629c394319bdb4c45f79da30f1:367:21
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3061:26
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
fireCallbacks@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3057:37
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3137:18
fireCallbacks@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1897:20
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1894:24
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
_fireCallbacks@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1893:25
_createView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1911:23
_materializeView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1915:20
visitObject@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1513:44
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:118:36
doMaterialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1933:48
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
doRender@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1931:26
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
with:materialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
Computation@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:224:18
autorun@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:499:34
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1831:29
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1922:17
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
_materializeView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1921:22
visitObject@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1513:44
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:118:36
visitArray@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1416:17
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:116:37
doMaterialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1933:48
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
doRender@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1931:26
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
if:materialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
Computation@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:224:18
autorun@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:499:34
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1831:29
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1922:17
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
_materializeView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1921:22
visitObject@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1513:44
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:118:36
visitArray@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1416:17
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:116:37
visitTag@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1495:41
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:103:37
visitArray@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1416:17
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:116:37
doMaterialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1933:48
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
doRender@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1931:26
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
if:materialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
Computation@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:224:18
autorun@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:499:34
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1831:29
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1922:17
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
_materializeView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1921:22
visitObject@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1513:44
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:118:36
visitArray@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1416:17
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:116:37
doMaterialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1933:48
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
doRender@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1931:26
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
with:materialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
Computation@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:224:18
autorun@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:499:34
autorun@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1831:29
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1922:17
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
_materializeView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1921:22
visitObject@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1513:44
visit@http://localhost:4000/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:118:36
doMaterialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1933:48
nonreactive@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:525:13
doRender@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1931:26
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1821:22
_withTemplateInstanceFunc@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:16
http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1820:54
_withCurrentView@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:16
if:materialize@http://localhost:4000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1819:34
_compute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:308:36
_recompute@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:322:22
flush@http://localhost:4000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:452:24
```

**Comment by JackAdams** [comment]
Weird. I haven't changed anything in the last week. It might be to do with api changing in 1.0.4.2? I'll look into it asap.

**Comment by msavin** [comment]
I doubt it. It does look like Mongol was using x.x.3 or something until I updated it. I'd guess that's where it began

**Comment by JackAdams** [comment]
FYI -- I've just released 0.3.11 with the only change being updated dependencies.  I don't think it will help.  I haven't had time to reproduce the error yet.  Will get back to you as soon as I do.

**Comment by JackAdams** [comment]
I've been unable to reproduce this error.  If there's a repo with a minimal reproduction I could look at, that would help.
